src/commands/apply.js

```diff
@@ -0,0 +1,101 @@
+const log = require('@dhis2/cli-helpers-engine').reporter
+const fs = require('fs-extra')
+const path = require('path')
+
+const { availableCodemods: availableCodemodsLocal } = require('../utils/codemods/availableCodemods')
+const { exec } = require('./apply/exec')
+const { findAvailableCodemodsInNodeModules } = require('../utils/codemods/findAvailableCodemodsInNodeModules')
+const { getCodemodByPackageAndName } = require('../utils/codemods/getPathByPackageAndName')
+const { mergeCodemods } = require('../utils/codemods/mergeCodemods')
+
+module.exports.command = 'apply <files..>'
```

Comment:
I propose this pattern, and that `<codemod>` is a delimited `package:name` string:

```
module.exports.command = 'apply <codemod> [files..]'
```